knative-operator/deploy/crds/operator_v1alpha1_knativekafka_crd.yaml

```diff
@@ -44,7 +47,6 @@ spec:
                     type: boolean
                 required:
                 - enabled
-                - bootstrapServers
```

Comment:
We already have a check in webhook. We reject the CR if `spec.channel.enabled=true` and `bootstrapServers` is empty